tests/integration/security/testdata/authz/v1beta1-deny.yaml.tmpl

```diff
@@ -51,3 +51,39 @@ spec:
     - operation:
         paths: ["/allow*"]
 ---
+
+# The following policy denies access to path /allow/admin to workload vm
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-vm-deny
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "vm"
```

Comment:
Why not begin to template our configs? Seems like this will create more work to be done later.

Reply:
+1, here and all other places, it seems we are just duplicating the yaml for VM, I would prefer to reuse and only keep one copy of the yaml, you can template it so that it can be used for both workloads.

Reply:
I have revised all the tests and templatized security policy files. Please take a look.